Formula/adios2.rb

```diff
@@ -0,0 +1,126 @@
+class Adios2 < Formula
+  desc "Next generation of ADIOS developed in the Exascale Computing Program"
+  homepage "https://adios2.readthedocs.io"
+  url "https://github.com/ornladios/ADIOS2/archive/v2.5.0.tar.gz"
+  sha256 "7c8ff3bf5441dd662806df9650c56a669359cb0185ea232ecb3578de7b065329"
+  head "https://github.com/ornladios/ADIOS2.git", :branch => "master"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "bzip2"
+  depends_on "c-blosc"
+  depends_on "catch2"
+  depends_on "hdf5"
+  depends_on "libfabric"
```

Comment:
@ax3l my understanding is that libfabric is only called if SST is ON

Reply:
yes, SST is on here:
```
    Build Options:
      Blosc    : ON
      BZip2    : ON
      ZFP      : OFF
      SZ       : OFF
      MGARD    : OFF
      PNG      : ON
      MPI      : OFF
      DataMan  : ON
      Table    : OFF
      SSC      : ON
      SST      : ON
      DataSpaces: OFF
      ZeroMQ   : ON
      HDF5     : ON
      Python   : ON
      Fortran  : ON
      SysVShMem: ON
      Profiling: ON
      Endian_Reverse: OFF
    RDMA Transport for Staging: Availabl
```